Dockerfile

```diff
@@ -1 +1,22 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_cloudconfig \
+      MAINTAINER="Armedia LLC"
+ARG RESOURCE_PATH="artifacts" 
+ENV CLOUD_CONFIG_VERSION="2021.03"
+WORKDIR /app
+RUN useradd --create-home --user-group arkcase \
+        && mkdir /app/tmp \
+        && chown -R arkcase:arkcase /app 
+
+USER arkcase
+#COPY the application war files
+COPY --chown=arkcase ${RESOURCE_PATH}/config-server-${CLOUD_CONFIG_VERSION}-SNAPSHOT.jar /app/config-server.jar 
```

Comment:
please combine these two COPY lines into one COPY

Reply:
@JonHolman  we cannot combine two COPY instructions  into one here. As in the first COPY we are copying to the file to the destination with different name `config-server.jar` .